From 989ec788adfd89e7b1f6a99102689ccea6ed95c6 Mon Sep 17 00:00:00 2001
From: Mitch Luban <mluban@nvidia.com>
Date: Thu, 31 Jan 2013 13:14:31 -0800
Subject: [PATCH 06/10] power: Add POWER_HINT_HIGH_RES_VIDEO to power hints

Need POWER_HINT_HIGH_RES_VIDEO to boost on 4k video
playback.

Bug 1185882

Reviewed-on: http://git-master/r/196152
(cherry picked from commit 40d7a59f82dc2ecb4794a803b3dc6e97f27709e5)

Change-Id: I6d761dc2bc7674d754fbbfca629ad4e324bea532
Reviewed-on: http://git-master/r/203511
Reviewed-by: Automatic_Commit_Validation_User
GVS: Gerrit_Virtual_Submit
Reviewed-by: Mitch Luban <mluban@nvidia.com>
Tested-by: Mitch Luban <mluban@nvidia.com>
Reviewed-by: Bharat Nihalani <bnihalani@nvidia.com>
---
 include/hardware/power.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/hardware/power.h b/include/hardware/power.h
index cc191bc..a62a751 100644
--- a/include/hardware/power.h
+++ b/include/hardware/power.h
@@ -44,7 +44,8 @@ typedef enum {
     POWER_HINT_APP_PROFILE = 0x00000003,
     POWER_HINT_APP_LAUNCH = 0x00000004,
     POWER_HINT_SHIELD_STREAMING = 0x00000005,
-    POWER_HINT_COUNT = 0x00000006,
+    POWER_HINT_HIGH_RES_VIDEO = 0x00000006,
+    POWER_HINT_COUNT = 0x00000007,
 } power_hint_t;
 
 /*
-- 
1.8.1.5

